libs/cart/testing/src/drivers/testing/cart.service.ts

```diff
@@ -24,4 +24,8 @@ export class DaffTestingCartService implements DaffCartServiceInterface<DaffCart
   clear(): Observable<void> {
     return of();
   }
+
+  create() {
+    return this.get();
```

Comment:
Let's just return a:

```
{
 id: this.cartFactory.create().id
}
```

as that will likely be the most common case.

Reply:
Changed in recent push.